ribs/_utils.py

```diff
@@ -0,0 +1,27 @@
+"""Miscellaneous internal utilities."""
+
+
+def check_measures_batch_shape(measures_batch,
+                               measure_dim,
+                               name="measures_batch"):
```

Comment:
You have a `name` parameter here but not for the `check_measures_shape` function

Reply:
Added name to `check_measures_shape`